.github/workflows/pull-request-closed.yml

```
@@ -0,0 +1,50 @@
+# https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#force-deleting-cache-entries
+name: Pull resquest closed
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cleanup up branch cache
+        run: |
+          gh extension install actions/gh-actions-cache
+
+          echo "Fetching list of cache key"
```

```suggestion
          echo "Fetching list of cache keys"
```
Should this be plural? (keys)? sorry super small comment just thought I'd ask!